### ISSUE TITLE
NameError: undefined local variable or method `id'

### ISSUE THREAD

**Issue by eftikharEmad** [Bandwidth/ruby-bandwidth-iris#23]
Hello,
I have simply this code 
```
   order = BandwidthIris::Order.get(id)
   order.methods
   order.get_tns
```
this will list array -> [...:get_notes, :add_notes, :get_history, :get_area_codes, :get_npa_npx, :get_totals, :get_tns,.....]
 but calling `get_tns` on the order object gives me `NameError: undefined local variable or method id for #<BandwidthIris::Order......`

am I missing something ?? 
thank you

**Comment by eftikharEmad** [comment]
this methods is only valid (working) for the returned created order!!
` 
order = BandwidthIris::Order.create(order_params)
 order.methods
 order.get_tns
`
working perfectly

**Comment by jmulford-bw** [comment]
Hi @eftikharEmad,

So it looks like this function was not updated when the live API changed. The API call for `BandwidthIris::Order.create(order_params)` does return an `id` value but `BandwidthIris::Order.get(id)` does not return an `id` value according to our API docs, which is why you're getting `undefined local variable or method id` when getting an order by the ID.

If you're completely blocked and need functionality right now, you can do a GET request to `https://dashboard.bandwidth.com/api/accounts/{accountId}/orders/{orderId}/tns` and you'll get a response like

```
<TelephoneNumbers>
    <Count>1</Count>
    <TelephoneNumber>8042105666</TelephoneNumber>
</TelephoneNumbers>
```

Otherwise we can publish a fix (maybe sometime this week)

Our API docs for number management can be found at https://dev.bandwidth.com/numbers/apiReference.html. These endpoints specifically can be found underneath `/accounts /{accountId} /orders`

**Comment by jmulford-bw** [comment]
Hey @eftikharEmad 

Please update to version `2.3.0` and use the method `BandwidthIris::Order.get_tns_by_order_id("id")` to grab tns for an already existing order

**Comment by eftikharEmad** [comment]
Hello @jmulford-bw 
thank you so much for replying, and I am sorry I was not able to answer sooner.

I am not completely blocked since the order object returned from order creating worked fine, so I used it, unless the order object does not present the order status 100%, I mean if there was any issue in allocating a number then the order object should reflect this, thats what I was trying to do with getting the order and check the `tns`.

thank you

**Comment by dtolb** [comment]
Also resolved in [#27](https://github.com/Bandwidth/ruby-bandwidth-iris/pull/27/files) natively by adding back the ID to the order object
